src/App.js

```diff
@@ -8,11 +8,13 @@ this file. If not, please write to: featurehuntteam@gmail.com
 import './App.css';
 import {BrowserRouter as Router, Switch, Route} from 'react-router-dom';
 import Product from './Components/Product';
+import ProjectForm from './Components/ProjectForm';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/CSC510-Group-25/feature-hunt/pullRequest?prid=8358184)
```suggestion
import ProjectForm from "./Components/ProjectForm";
```